doc/api/dns.md

```diff
@@ -417,6 +418,51 @@ is a two-dimensional array of the text records available for `hostname` (e.g.,
 one record. Depending on the use case, these could be either joined together or
 treated separately.
 
+## dns.resolveAny(hostname, callback)
+
+- `hostname` {string}
+- `callback` {Function}
+  - `err` {Error}
+  - `ret` {Object[][]}
+
+Uses the DNS protocol to resolve any queries (`ANY` records) for the `hostname`.
+The `ret` argument passed to the `callback` function will be an array of objects
+with uncertain type of records. Each object has a property `type` that indicates
+the type of current record. And for each type of record, the object structure
```

Comment:
Maybe replace the last sentence with: "Depending on the `type`, additional properties will be present on the object: